bittensor/_dendrite/dendrite_impl.py

```diff
@@ -351,6 +351,121 @@ def _forward(
 
         return packed_outputs, packed_codes, packed_times
 
+
+    def generate(
```

Comment:
What's the intention of this function??

It's something to be used inside bittensor code / python code. I'm wondering wether exposing an API would have sense